crates/teloxide-core/src/types/message.rs

```diff
@@ -1090,73 +1098,85 @@ mod getters {
         // TODO: OK, `Option<True>` is weird, can we do something with it?
         //       mb smt like `is_delete_chat_photo(&self) -> bool`?
         #[must_use]
-        pub fn delete_chat_photo(&self) -> Option<True> {
+        pub fn delete_chat_photo(&self) -> Option<&True> {
```

Comment:
What is the reasoning behind this change?

Reply:
The macro define_message_ext! uses ToOwned::to_owned in it's body, so when I try to use an existing getters which doesn't return Option<&some_type> I receive an error(tied with the to_owned, as I understood correctly) 

I don't want to duplicate existing code with similar functions names, so, decided to change slightly the signature of some  getters (in the same way as in this function)

Reply:
I don't think that's right. If you use `filter_delete_char_photo`, you really want `MessageChannelChatCreated`, not `True` (which is ambiguous, useless, etc).

I guess this should be changed to return `Option<&MessageChannelChatCreated>` & then we should have `fn is_delete_char_photo(&self) -> bool`?